app/services/mapping/governmentagencygoodsitem/AdditionalDocumentsBuilder.scala

```diff
@@ -19,14 +19,23 @@ package services.mapping.governmentagencygoodsitem
 import forms.declaration.additionaldocuments.DocumentsProduced
 import javax.inject.Inject
 import models.declaration.governmentagencygoodsitem._
-import services.ExportsItemsCacheIds.dateTimeCode
 import services.cache.ExportItem
 import services.mapping.ModifyingBuilder
 import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment
 import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment.GovernmentAgencyGoodsItem.AdditionalDocument
-import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment.GovernmentAgencyGoodsItem.AdditionalDocument.{Submitter, WriteOff => WCOWriteOff}
-import wco.datamodel.wco.declaration_ds.dms._2.AdditionalDocumentEffectiveDateTimeType.{DateTimeString => WCODateTimeString}
-import wco.datamodel.wco.declaration_ds.dms._2.{AdditionalDocumentEffectiveDateTimeType, SubmitterNameTextType, WriteOffQuantityQuantityType, _}
+import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment.GovernmentAgencyGoodsItem.AdditionalDocument.{
+  Submitter,
+  WriteOff => WCOWriteOff
+}
+import wco.datamodel.wco.declaration_ds.dms._2.AdditionalDocumentEffectiveDateTimeType.{
+  DateTimeString => WCODateTimeString
+}
+import wco.datamodel.wco.declaration_ds.dms._2.{
+  AdditionalDocumentEffectiveDateTimeType,
+  SubmitterNameTextType,
+  WriteOffQuantityQuantityType,
+  _
```

Comment:
If we're importing at the end everything, do we need to have three types specified before?

Reply:
This will be intelliJ doing its thing as I cleaned up the imports as I had unused ones.